### ISSUE TITLE
Skull Data & Armor Stands

### ISSUE THREAD

**Issue by TylerS1066** [msummers123/Movecraft-3#58]
Skull data is messed up when a ship is moved

Before:
![2014-11-26_19 22 54](https://cloud.githubusercontent.com/assets/7319776/5211191/c5ee5e2c-75a1-11e4-884d-14e1dcdeac3b.png)

After:
![2014-11-26_19 23 27](https://cloud.githubusercontent.com/assets/7319776/5211194/ce854b9a-75a1-11e4-90f0-2a07ae9ffc95.png)

**Comment by TylerS1066** [comment]
Are armor stands as tricky as skulls?

**Comment by msummers123** [comment]
Actually I'm afraid armor stands are much worse, because they aren't even blocks but rather entities. Yeah, I don't think we will see armor stand support anytime soon.

**Comment by TylerS1066** [comment]
Sinced they are entities doesn't the ship move them?

**Comment by msummers123** [comment]
Yeah, it should teleport them. Try it and see, but I suspect that like other entities (paintings, minecarts) it will work some times, but randomly fail at other times.

**Comment by Pocketkid2** [comment]
Can't you just copy the extra data in the same way that you copy sign text and rotation?

**Comment by keough99** [comment]
You could maybe use a bit of the programming from the signs and chests? If you didn't know signs and chests are actually considered entities they go in the same category as entities but they have their own entity family they are considered block entities and skulls from what I know fall in the same entity family amor stands the data would need to be copied. And what about banner support?
